exercises/complex-numbers/complex_numbers_test.cpp

```diff
@@ -0,0 +1,218 @@
+#include "complex_numbers.h"
+
+#include <cmath>
+#include <complex>
+
+#include "test/catch.hpp"
+
+// Define M_PI for platforms that don't include it
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
+
+// Complex Numbers exercise test case data version 1.0.0
```

Comment:
I've been trying to add the version info to the test suites when they get touched, since it's easier to tell if it needs an update.

(If this comment was talking about defining `M_PI`, it isn't defined on Windows for some reason. It's supposed to be [1], but that didn't work for me so defining it ourselves is easier than fixing compat problems.)

[1] https://docs.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=vs-2019

Reply:
Ah, my comment was unclear; I meant to refer to the version number.

I didn't realize that you added in the version info - I thought it might've been carried over from the original commit in 2017. Does the version number need to sync with something specific (say, the problem-specifications repo) or do we just increment it as we touch the file?

About `M_PI`, defining it manually is the safer route. Who knows what other compilers will do....